common/config/rush/pnpm-lock.yaml

```diff
@@ -21201,3 +21201,4 @@ specifiers:
   '@rush-temp/react-composites': file:./projects/react-composites.tgz
   '@rush-temp/server': file:./projects/server.tgz
   '@rush-temp/storybook': file:./projects/storybook.tgz
+
```

Comment:
Intentional? Or whitespace change to test your PR?

Reply:
Yeah intentional, I was hoping it would cache bust but I guess a single blank new line in a 21000 line file didn't make enough of a difference 😅 (still got the old cache for some of the jobs)

Reply:
It should cache bust. Perhaps github's build cache is distributed and the first job actually helped out the ones that started later? Or is it just a local cache on a persistent VM?

Reply:
It only uploads the cache at the end of the job, however some jobs definitely grabbed from a cache when they shouldn't of given the change to this file.. don't have time to look into it right now but I'll leave this PR open until I do

Reply:
Fixed the cache key, had `\`s instead of `/`s in the glob path 🤦‍♂️

Reply:
> > You can shave off another 37 seconds if you also cache `common/temp/node_modules`.
> > Should be safe to cache with the same key? After all the node_modules are just decompressed result from the pnpm store.
> > wdyt?
> 
> I think we should do this 👍

Caching node_modules had no impact on the time for rush install, but did increase the restore from cache by ~6s so removing